EMacco/Customer-Ticketing-System
# #168722491 Login user

#### What does this PR do?
This PR allows a user to be able to log into the system

#### Description of Task to be completed?
- Create a route `POST /auth/login`
- Validate user details
- Add user details to the session
- Return the response
- Write tests

#### How should this be manually tested?
- Checkout to this branch
- Make a post request to `POST /auth/login` with email and password
- You should receive either an object of the user or an error message if the credentials are invalid

#### What are the relevant pivotal tracker stories?
[#168722491](https://www.pivotaltracker.com/story/show/168722491)
